packages/dashboard/src/dashboard-actions/dashboardReducer.ts

```diff
@@ -34,10 +35,11 @@ export const dashboardReducer: Reducer<DashboardState, DashboardAction> = (
         return {
           ...state,
           selectedWidgetIds: action.payload.widgetIds,
+          previousPosition: action.payload.position,
           intermediateDashboardConfiguration: move({
             dashboardConfiguration: state.intermediateDashboardConfiguration || state.dashboardConfiguration,
             position: action.payload.position,
-            previousPosition: action.payload.prevPosition,
+            previousPosition: state.previousPosition || action.payload.position,
```

Comment:
if there is no previous position, shouldn't it be `undefined`?

Reply:
if it is left undefined, it kind of jumps all over the place when you try to move it. this is because of how the move() is written. i'm not sure if it's a bug, but if previous position doesn't exist, the delta returned is the current position rather than the delta being 0. So the way to fix that bug is either include action.payload.position, or i can edit your code. i just wasn't sure if you wanted me to touch move